packages/components/src/components/hds/alert/index.ts

```diff
@@ -40,7 +40,7 @@ export interface HdsAlertSignature {
   Args: {
     type: HdsAlertTypes;
     color?: HdsAlertColors;
-    icon?: FlightIconSignature['Args']['name'] | false;
+    icon?: HdsIconSignature['Args']['name'] | false;
```

Comment:
I am seeing a number of variations like `boolean | undefined` would it make sense to expand `HdsIconSignature` @alex-ju @didoo ?

Reply:
I will assess each instance as I review the code but, if my memory serves me well, only the Alert component should have this option of false (as we want to provide consumers with a way to suppress the default icon, if needed).